examples/structure_manager_iteration.cc

```diff
@@ -61,16 +61,16 @@ int main() {
    *
    * `crystal_structure.json` is a fully periodic metallic hcp structure.
    *
-   * `alanine-X.json` is a unit cell of a polyalanine chain.
+   * `../reference_data/alanine-X-examples.json` is a unit cell of a polyalanine chain.
```

Comment:
Probably don't need to give the whole path here if it's given below, and if it's understood from now on that atomic structure files live in `reference_data/inputs/`